scout/ecm_prep.py

```diff
@@ -13764,6 +13901,38 @@ def main(opts: argparse.NameSpace):  # noqa: F821
                 meas_toprep_package, meas_prepped_objs, meas_summary,
                 handyvars, handyfiles, base_dir, opts, convert_data)
 
+        # Warn users about skipped ECMs before completing prep execution
+        if len(handyvars.skipped_ecms) != 0:
+            # Calculate relative share of skipped ECMs for warning
+            rel_share = "(" + str(int((len(handyvars.skipped_ecms) / (
+                len(meas_toprep_indiv) +
+                len(meas_toprep_package))) * 100)) + "%)"
+            # Set singular or plural use of ECMs for error message
+            if len(handyvars.skipped_ecms) == 1:
+                single_plural = " ECM " + rel_share + " was "
+            else:
+                single_plural = " ECMs " + rel_share + " were "
+            # Warn user
+            warnings.warn(
+                "WARNING: " +
+                str(len(handyvars.skipped_ecms)) + single_plural +
+                "not prepared due to exceptions. See TXT file with the "
+                "appropriate timestamp in ./logs for details.")
```

Comment:
So could there be multiple log files during a single run of ecm_prep.py? I though they would all be in one file